tree/dataframe/src/RCsvDS.cxx

```diff
@@ -16,14 +16,17 @@
 The RCsvDS class implements a CSV file reader for RDataFrame.
 
 A RDataFrame that reads from a CSV file can be constructed using the factory method
-ROOT::RDF::MakeCsvDataFrame, which accepts three parameters:
+ROOT::RDF::MakeCsvDataFrame, which accepts five parameters:
 1. Path to the CSV file.
 2. Boolean that specifies whether the first row of the CSV file contains headers or
 not (optional, default `true`). If `false`, header names will be automatically generated as Col0, Col1, ..., ColN.
 3. Delimiter (optional, default ',').
+4. Chunk size (optional, default is -1 to read all) - number of lines to read at a time
+5. Column Types (optional, default is an empty map). A map with column names as keys and their type
+(expressed as a single character, see below) as values.
+The type of columns that do not appear in the map is inferred from the data.
 
-The types of the columns in the CSV file are automatically inferred. The supported
-types are:
+The supported types are:
```

Comment:
should we mention the corresponding characters 'O', 'L' etc here? I'm not sure we document them for users anywhere